test/end-to-end/util/dummyFile.txt

```diff
@@ -0,0 +1 @@
+For testing file upload
```

Comment:
also an additional point for my 11MB file comment: if we are able to dynamically generate files and use them in `setFilesToUpload` then we can also delete this i presume